### DISCUSSION TITLE
识别模型v4_server转onnx推理非常慢

### DISCUSSION THREAD

**Discussion by xiaoshuizhu202** [PaddlePaddle/PaddleOCR#12240] [Q&A]
模型转成功了，识别的推理需要10分钟，请问是什么原因？

**Comment by xxdm** [comment]
我也很慢，我也需要10多分钟

**Comment by xiaoshuizhu202** [comment]
好奇怪，gpu上时间正常的





---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年9月27日(周三) 晚上11:38
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [PaddlePaddle/PaddleOCR] 识别模型v4_server转onnx推理非常慢 (Issue #10797)




 
我也很慢，我也需要10多分钟

**Comment by SWHL** [comment]
可以先推理一下模型转换前格式，用Paddle在CPU上推理看看，sever版可能本来就很慢
